addons/mail/static/src/discuss/core/public_web/thread_actions.js

```diff
@@ -30,10 +30,13 @@ threadActionsRegistry.add("show-threads", {
             },
         });
     },
-    open: (component, action) =>
+    open: (component, action) => {
+        const thread = component.thread?.parent_channel_id || component.thread;
         action.popover?.open(component.root.el.querySelector(`[name="${action.id}"]`), {
-            thread: component.thread,
-        }),
+            thread,
+            close: action.close(),
```

Comment:
```suggestion
            close: () => action.close(),
```
I might be mistaken but I think you want to pass the close function not invoke it? Also are you sure we need to pass it? It looks like it is overwritten by `close` function that is passed to `overlayService.add` by the `popoverService`?

Reply:
You are right, we do not need to pass the close function here.